custom/conf/app.ini.sample

```diff
@@ -145,6 +145,15 @@ CUSTOM_URL_SCHEMES =
 ; List of file extensions that should be rendered/edited as Markdown
 ; Separate the extensions with a comma. To render files without any extension as markdown, just put a comma
 FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
+; Create Table of Contents based on headlines on wiki page (does not alter file)
+DEFAULT_TOC_WIKI_FILE = true
+; Create Table of Contents on all rendered markdown files (does not alter file), enabled: overwrites DEFAULT_TOC_MARKDOWN_BY_FLAG
+DEFAULT_TOC_MARKDOWN_ALWAYS = false
+; Create Table of Contents on renderes markdown files if line '%%TOC%%' is present
+DEFAULT_TOC_MARKDOWN_BY_FLAG = true
+; List of markup file extensions that TOC should be created on, set to not interfere with external renderers
+; Separate the extensions with a comma. To ignore file extension check, just put a comma
+TOC_MARKDOWN_FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd,.org
```

Comment:
Maybe `MARKUP` is better since not all of them are markdown format.

Reply:
Done. 
renamed 'markdown' to 'markup' in app.ini and variables